tests/0113-cooperative_rebalance.cpp

```diff
@@ -257,79 +369,1698 @@ void run_test(std::string &t1, std::string &t2,
     delete consumer;
 }
 
-extern "C" {
-  int main_0113_cooperative_rebalance (int argc, char **argv) {
+
+static void a_assign_tests () {
+    Test::Say("Executing a_assign_tests\n");
+
     int msgcnt = 1000;
     const int msgsize1 = 100;
     const int msgsize2 = 200;
 
     std::string topic1_str = Test::mk_topic_name("0113-cooperative_rebalance", 1);
     test_create_topic(NULL, topic1_str.c_str(), 1, 1);
-    test_produce_msgs_easy_size(topic1_str.c_str(), 0, 0, msgcnt, msgsize1);
-
     std::string topic2_str = Test::mk_topic_name("0113-cooperative_rebalance", 1);
     test_create_topic(NULL, topic2_str.c_str(), 1, 1);
+    sleep(TOPIC_CREATE_WAIT_S);
```

Comment:
ahh, i should have read these comments first, i added this to test_create_topic. but this is better because using the client instance used in the rest of the test for the metadata request will make it robust.